docs/tutorials/configure/scuttlebot.md

```diff
@@ -159,164 +240,185 @@ This will be similar to adding the right motor.
       "pwm": ""
     },
     "board": "local",
-    "max_rpm": 100,
-    "dir_flip": false
+    "dir_flip": false,
+    "ticks_per_rotation": 2
   },
-  "depends_on": []
+  "depends_on": [ "local" ]
 }
 ```
 
-{{% /expand %}}
+{{% /tab %}}
+{{< /tabs >}}
 
-With both motors configured, the **Control** tab now display panels for both motors:
+Save the config by clicking **Save config** at the bottom of the page.
+### Test the motor configuration
+
+Now that you have configured your board, encoders, and motors, you can actuate your motors.
+Navigate to the **Control** tab.
+
+You'll see a panel for each configured component.
 
 ![Motor panels](/tutorials/scuttlebot/scuttle-bothmotors.png)
 
-Viam ([app.viam.com](https://app.viam.com)) displays component panels in order of their creation.
-Don't worry if your motor panels are not adjacent.
+Click on the panel for the right `motor`.
+
+![Power level adjustment](/tutorials/scuttlebot/pi-moverhmotor.png)
+
+Try changing the motor's **power** level and click **Run**.
 
-## Configuring the base
+{{< alert title="Caution" color="caution" >}}
+Be careful when using your motors!
+Start with the power level set to 10% and increase it incrementally until the wheel rotates at a reasonable speed, clicking **Run** at each increment.
```

Comment:
```suggestion
Start with the power level set to 20% and increase it incrementally until the wheel rotates at a reasonable speed, clicking **Run** at each increment.
```
[opt] I think 10% is unnecessarily low and folks will indeed most likely have a whining if they start here.